### PR TITLE
Writing Day: add Raspberry Pi to project list

### PR DESCRIPTION
**Pull request by rosewms**
Edited version of Raspberry Pi's Writing Day submission.

----
:books: Documentation preview :books:: https://writethedocs-www--1937.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2023/writing-day.rst [hunk 1]
```diff
@@ -32,6 +32,30 @@ Project Listing
...
+
+Our online documentation is marked up in AsciiDoc, lives in Git, and is built automatically into 
+a static site using GitHub Actions.
+
+Raspberry Pi is looking for Writing Day attendees to contribute to our open source documentation. 
+We're looking for contributions that focus on: 
+
+- Copy-editing
+- Narrative structure
+-  `Style Guide <https://github.com/raspberrypi/style-guide>`_ improvments
```
Comment (plaindocs) [inline docs/conf/portland/2023/writing-day.rst]:
```suggestion
-  `Style Guide <https://github.com/raspberrypi/style-guide>`_ improvements
```

**Review by plaindocs — APPROVED** [review]
:shipit: .

**Comment by donatocayurin** [comment]
Thank you ❤️🙏

**Comment by donatocayurin** [comment]
❤️🙏

**Pull request merged**